### ISSUE TITLE
Can't drag file to the project directly

### ISSUE THREAD

**Issue by hbwang85** [decosoftware/deco-ide#87]
I think it will be very useful if we can drag file to the project directly, especially for the assets.  Hope this feature could be added in the future.

**Comment by dabbott** [comment]
Good idea. I'm working on a better file tree at the moment. After that, it should be easier and more clear how to add file dnd.
